### DISCUSSION TITLE
DataTable: resizable columns but make selected columns non-resizable

### DISCUSSION THREAD

**Discussion by sschwandter** [primefaces/community#193] [PrimeVue] [answered]
Hello!

The title says it all: is it possible to selectively disable column resize for certain columns? I didn't find a corresponding column property.

Best regards,
Stefan

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello!

It can be done by adding `pt` prop to non-resizable Column.


```
const isResizable = ref(false);


<Column  field="code" header="Code"
    :pt="{
        columnresizer: {
            class: [
              {hidden: !isResizable.value}
           ],
       },
   }"
></Column>
```

**Reply by sschwandter** [reply]
Great, thank you for your quick reply!

How does this play with saved state, though? It seems the width of that column is saved nevertheless, and if there is a value for that column, it will be loaded. This is a problem if a user already has a saved state with a width that's too small or too big.

Is there a possibility to reset the saved state just for that one column each time the TableView is created?
